app/src/main/kotlin/com/simplemobiletools/keyboard/helpers/EmojiHelper.kt

```diff
@@ -59,3 +62,27 @@ fun parseRawEmojiSpecsFile(context: Context, path: String): MutableList<String>
     cachedEmojiData = emojis
     return emojis
 }
+
+
+fun parseRawJsonSpecsFile(context: Context, path: String): HashMap<String,String> {
+    if (cachedVNTelexData.isNotEmpty()) {
+        return cachedVNTelexData
+    }
+
+    try {
+        val  inputStream: InputStream = context.assets.open(path)
```

Comment:
use ctrl + alt + L to format text, also avoid writing double empty lines like amove